src/pool/srv_pool_map.c

```diff
@@ -261,30 +267,33 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 			break;
 		case PO_COMP_ST_UP:
 			if (target->ta_comp.co_fseq == 1) {
-				D_DEBUG(DB_MD, "change "DF_TARGET" to NEW %p\n",
-					DP_TARGET(target), map);
+				D_DEBUG(DB_MD, DF_MAP ": change " DF_TARGET " to NEW\n",
+					DP_MAP(pool_uuid, map), DP_TARGET(target));
 				target->ta_comp.co_status = PO_COMP_ST_NEW;
 				target->ta_comp.co_in_ver = 0;
 				++(*version);
 			} else {
-				D_DEBUG(DB_MD, "change "DF_TARGET" to DOWNOUT %p fseq %u\n",
-					DP_TARGET(target), map, target->ta_comp.co_fseq);
+				D_DEBUG(DB_MD, DF_MAP ": change " DF_TARGET " to DOWNOUT fseq %u\n",
+					DP_MAP(pool_uuid, map), DP_TARGET(target),
+					target->ta_comp.co_fseq);
 				if (target->ta_comp.co_flags & PO_COMPF_DOWN2UP)
 					target->ta_comp.co_status = PO_COMP_ST_DOWN;
 				else
 					target->ta_comp.co_status = PO_COMP_ST_DOWNOUT;
 				target->ta_comp.co_out_ver = ++(*version);
 			}
 			if (print_changes)
-				D_PRINT(DF_TARGET" is excluded.\n", DP_TARGET(target));
-			else
-				D_INFO(DF_TARGET" is excluded.\n", DP_TARGET(target));
+				D_PRINT(DF_MAP ": " DF_TARGET " is excluded.\n",
```

Comment:
This one is more difficult for me to interpret quickly, and then I suppose get the log message perfectly accurate ("excluded" not always the case?). It seems the state may have changed to NEW, DOWN, or DOWNOUT. Maybe not worth perfecting it right now (if my understanding is accurate to begin with).

Reply:
Changed to "reverted". Please double check.

Reply:
That seems good, though I think the change will be needed also in the D_PRINT() in addition to the D_INFO() call.

Reply:
Oops, sorry.